keyboards/pmk/posey_split/v4/keymaps/via/keymap.c

```diff
@@ -0,0 +1,73 @@
+// Copyright 2024 Javier Oliver (@joliverMI)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+
+//System Layers
+#define _QWERTY 0
+#define _NAV 8
+#define _SYMB 9
```

Comment:
This has been requested multiple times
```suggestion
enum my_layers {
    _QWERTY,
    _NAV,
    _SYMB,
};
```